mozilla/blurts-server
# Add element view telemetry (MNTOR-3358)

# References:

Jira: [MNTOR-3358](https://mozilla-hub.atlassian.net/browse/MNTOR-3358)



# Description

Adds the hook `useViewTelementry`, which reports a telemetry event when an item is shown within the viewport for the first time. For this hook, a new Glean module `view` with the event name `enter` has been added.

# How to test

1. (Re)build Glean with `npm run build-glean`
2. Visit the landing page `/` and scroll through the page
3. Observe the telemetry events
